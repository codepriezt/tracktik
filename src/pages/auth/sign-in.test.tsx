import React from "react"
import {shallow} from "enzyme"
import SignIn from "./sign-in"

describe('<signIn/> component' , () => {







it("render" , () => {
        expect(shallow(<SignIn/> )).toMatchSnapshot()
})
})