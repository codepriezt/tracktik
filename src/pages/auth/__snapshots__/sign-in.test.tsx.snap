// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<signIn/> component render 1`] = `
<div
  className="flex items-center justify-center w-full min-h-screen px-4 py-12 bg-gray-50 sm:px-6 lg:px-8"
>
  <div
    className="w-full max-w-md space-y-8"
  >
    <div>
      <img
        alt="Workflow"
        className="w-auto h-12 mx-auto"
        src="https://tailwindui.com/img/logos/workflow-mark-indigo-600.svg"
      />
      <h2
        className="mt-6 text-3xl font-extrabold text-center text-gray-900"
      >
        Sign in to TrackTik
      </h2>
    </div>
    <form
      className="mt-8 space-y-6"
      onSubmit={[Function]}
    >
      <input
        defaultValue="true"
        name="remember"
        type="hidden"
      />
      <div
        className="-space-y-px rounded-md shadow-sm"
      >
        <div
          className="mb-5"
        >
          <label
            className="sr-only"
            htmlFor="email-address"
          >
            Email address
          </label>
          <input
            autoComplete="email"
            className="relative block w-full px-3 py-2 text-gray-900 placeholder-gray-500 border border-gray-300 rounded-none appearance-none rounded-t-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm"
            id="email-address"
            name="email"
            onChange={[Function]}
            placeholder="Email address"
            required={true}
            type="email"
            value="tracktik@gmail.com"
          />
        </div>
        <div>
          <label
            className="sr-only"
            htmlFor="password"
          >
            Password
          </label>
          <input
            autoComplete="current-password"
            className="relative block w-full px-3 py-2 text-gray-900 placeholder-gray-500 border border-gray-300 rounded-none appearance-none rounded-b-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm"
            id="password"
            name="password"
            onChange={[Function]}
            placeholder="Password"
            required={true}
            type="password"
            value="password12345"
          />
        </div>
      </div>
      <div
        className="flex items-center justify-between"
      >
        <div
          className="flex items-center"
        >
          <input
            className="w-4 h-4 text-indigo-600 border-gray-300 rounded focus:ring-indigo-500"
            id="remember-me"
            name="remember-me"
            type="checkbox"
          />
          <label
            className="block ml-2 text-sm text-gray-900"
            htmlFor="remember-me"
          >
            Remember me
          </label>
        </div>
      </div>
      <div>
        <button
          className="relative flex justify-center w-full px-4 py-2 text-sm font-medium text-white bg-indigo-600 border border-transparent rounded-md group hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
          type="submit"
        >
          <span
            className="absolute inset-y-0 left-0 flex items-center pl-3"
          />
          Sign in
        </button>
      </div>
    </form>
  </div>
</div>
`;
